src/dashboard/components/add_dashboard_modal.py

```diff
@@ -0,0 +1,94 @@
+"""Modal for add dashboard buttons."""
+from dash import html
+
+from dashboard.components import modal
+from dashboard.components.button import button
+from dashboard.components.multiline_input import multiline_input
+from dashboard.components.text_input import text_input
```

Comment:
The new components should be exported in the components/__init__.py module and can then be imported on the same line as modal.

Reply:
It seems modules from the same directory can not make use of the init.py default exports.